egcg_core/config.py

```diff
@@ -27,7 +27,9 @@ def load_config_file(self, *search_path, env_var=None):
             self.env_var = env_var
         self.config_file = self._find_config_file(search_path)
         if self.config_file:
-            self.content = yaml.safe_load(open(self.config_file, 'r'))
+            with open(self.config_file, 'r') as f:
+                self.content = yaml.safe_load(f)
+
```

Comment:
Just out of interest, why did you split it in this way?

Reply:
Using the context manager `with __ as _:` automatically closes the file handle after the statement has completed - see #89